packages/host/app/services/matrix-service.ts

```diff
@@ -875,31 +888,65 @@ export default class MatrixService extends Service {
     }
   }
 
-  async loadDefaultSkills(submode: Submode) {
-    let interactModeDefaultSkills = [`${baseRealm.url}SkillCard/card-editing`];
-
-    let codeModeDefaultSkills = [
-      `${baseRealm.url}SkillCard/boxel-coding`,
-      `${baseRealm.url}SkillCard/source-code-editing`,
-    ];
+  async setAiSettings(aiSettings: AiSettingsCard) {
+    await this.client.setAccountData(APP_BOXEL_AI_SETTINGS_EVENT_TYPE, {
+      id: aiSettings.id,
+    });
+  }
 
-    let defaultSkills;
+  private async getAiSettings(): Promise<AiSettingsCard | undefined> {
+    // Check if the user has set default skills for this submode
+    if (this.aiSettingsCardId) {
+      let aiSettings = await this.store.peek<AiSettingsCard>(
+        this.aiSettingsCardId,
+      );
+      if (aiSettings) {
+        return aiSettings;
+      } else {
+        console.log('cannot load ai settings', this.aiSettingsCardId);
+      }
+    }
+    let { id: useraiSettingsCardId } =
+      (await this.client.getAccountDataFromServer<{
+        id: string;
+      }>(APP_BOXEL_AI_SETTINGS_EVENT_TYPE)) || { id: undefined };
+    if (!useraiSettingsCardId) {
+      // load the default
+      console.log('no user ai settings, loading default');
+      return undefined;
+    }
+    let aiSettings =
+      await this.store.peek<AiSettingsCard>(useraiSettingsCardId);
```

Comment:
Ideally I'd use the store to keep a reference that gets automatically updated instead of using peek